app.js

```diff
@@ -3,12 +3,17 @@ const hashString = 'Hello from IPFS Gateway Checker'
 
 const $results = document.querySelector('#results')
 
+function returnHtmlLink (gateway) {
+	let gatewayTitle = gateway.split(hashToTest)[0]
+	return '<a title="' + gatewayTitle + '" href="' + gateway '" >' + gateway + '</a>';
+}
+
 function addNode (gateway, online, title) {
 	const para = document.createElement('div')
 	let node
 	if (online) {
 		node = document.createElement('strong')
-		node.innerText = '✅ - Online  - ' + gateway
+		node.innerText = '✅ - Online  - ' + returnHtmlLink(gateway)
```

Comment:
I don't think this will work, `innerText` only sets text. Would have to change to be `innerHTML` here.

Reply:
indeed